### PR TITLE
Only configure setuptools logging if bdist_wheel is imported

### LINKED ISSUE #632: `wheel tags` command displays invalid filenames on stdout
To quote `python -m wheel tags --help`:

> The output filename(s) will be displayed on stdout for further processing.

This should make usage in shell scripts like `FIXED_WHEEL_PATH=$(python -m wheel tags ...)` (with single input file) possible.

However, instead, the stdout is spammed with "added: 'module/some/file.py'" lines, which are not valid filenames. Such logging should be sent to stderr instead.

<details>
<summary>Example output of `python -m wheel tags`</summary>

```terminal
$ python -m wheel tags --python-tag py3 --abi-tag none --platform-tag manylinux2014_x86_64.manylinux_2_17_x86_64 dist/impuls-0.6.0-cp312-cp312-linux_x86_64.whl 2>/dev/null
adding 'impuls-0.6.0.dist-info/METADATA'
adding 'impuls-0.6.0.dist-info/WHEEL'
adding 'impuls/extern/libextern.so'
adding 'impuls/extern/__init__.py'
adding 'impuls/model/meta/__init__.py'
adding 'impuls/model/meta/entity.py'
adding 'impuls/model/meta/sql_builder.py'
adding 'impuls/model/meta/utility_types.py'
adding 'impuls/model/__init__.py'
adding 'impuls/model/agency.py'
adding 'impuls/model/attribution.py'
adding 'impuls/model/calendar_exception.py'
adding 'impuls/model/calendar.py'
adding 'impuls/model/fare_attribute.py'
adding 'impuls/model/fare_rule.py'
adding 'impuls/model/feed_info.py'
adding 'impuls/model/frequency.py'
adding 'impuls/model/route.py'
adding 'impuls/model/shape_point.py'
adding 'impuls/model/stop.py'
adding 'impuls/model/stop_time.py'
adding 'impuls/model/transfer.py'
adding 'impuls/model/translation.py'
adding 'impuls/model/trip.py'
adding 'impuls/tasks/__init__.py'
adding 'impuls/tasks/add_entity.py'
adding 'impuls/tasks/exec_sql.py'
adding 'impuls/tasks/generate_trip_headsign.py'
adding 'impuls/tasks/load_busman.py'
adding 'impuls/tasks/load_gtfs.py'
adding 'impuls/tasks/merge.py'
adding 'impuls/tasks/modify_from_csv.py'
adding 'impuls/tasks/remove_unused_entities.py'
adding 'impuls/tasks/save_db.py'
adding 'impuls/tasks/save_gtfs.py'
adding 'impuls/tasks/truncate_calendars.py'
adding 'impuls/tools/__init__.py'
adding 'impuls/tools/color.py'
adding 'impuls/tools/geo.py'
adding 'impuls/tools/iteration.py'
adding 'impuls/tools/logs.py'
adding 'impuls/tools/machine_load.py'
adding 'impuls/tools/polish_calendar_exceptions.py'
adding 'impuls/tools/strings.py'
adding 'impuls/tools/temporal.py'
adding 'impuls/tools/testing_mocks.py'
adding 'impuls/tools/types.py'
adding 'impuls/__init__.py'
adding 'impuls/app.py'
adding 'impuls/db.py'
adding 'impuls/errors.py'
adding 'impuls/multi_file.py'
adding 'impuls/options.py'
adding 'impuls/pipeline.py'
adding 'impuls/py.typed'
adding 'impuls/resource.py'
adding 'impuls/task.py'
adding 'impuls-0.6.0.dist-info/RECORD'
impuls-0.6.0-py3-none-manylinux2014_x86_64.manylinux_2_17_x86_64.whl
```
</details>

### PR DESCRIPTION
**Pull request by agronholm**
This prevents the CLI commands from emitting unwanted output.

Fixes #632.

### CODE REVIEW

File: src/wheel/_bdist_wheel.py [hunk 1]
```diff
@@ -34,6 +34,15 @@
 if TYPE_CHECKING:
     import types
 
+# ensure Python logging is configured
+try:
+    __import__("setuptools.logging")
```
Comment (henryiii) [inline src/wheel/_bdist_wheel.py:39]:
```suggestion
    import setuptools.logging
```

Why not this? They should be the same, right? Usually you use this form if you have a variable value to import.

Comment (henryiii) [reply src/wheel/_bdist_wheel.py:39]: (Though now I see this is just moved...)

File: src/wheel/cli/convert.py [hunk 2]
```diff
@@ -293,7 +293,7 @@ def convert(files: list[str], dest_dir: str, verbose: bool) -> None:
                 source = WininstFileSource(path)
 
             if verbose:
-                print(f"{archive}... ", flush=True)
+                print(f"{archive}...", flush=True, end="")
```
Comment (henryiii) [inline src/wheel/cli/convert.py:296]: Should this be to stderr?

**Review by henryiii — APPROVED** [review]

File: src/wheel/cli/convert.py (hunk 2 above)
Comment (agronholm) [reply src/wheel/cli/convert.py:296]: I don't think so...? There's no docs indicating either way, and `wheel tags` prints the file names to stdout.

**Pull request merged**